testing/api/test_gui.py

```diff
@@ -5,14 +5,15 @@
 import spinalcordtoolbox.image as msct_image
 from spinalcordtoolbox.gui import base, centerline, sagittal
 from spinalcordtoolbox.gui.base import InvalidActionWarning
+from spinalcordtoolbox.utils import sct_test_path
 
 
 logging.basicConfig(level=logging.DEBUG)
 
 
 class CenterlineTestCase(unittest.TestCase):
```

Comment:
Ah oops, of course. These tests [can't run in our CI](https://github.com/spinalcordtoolbox/spinalcordtoolbox/runs/3048345891#step:5:30) because they're GUI tests. :facepalm: 

Still though, all of the testing is done programmatically, so I wonder if there's a way to change the `PyQt` backend to allow for testing anyway.

Reply:
Oh, wait a minute. The CI error is from a `matplotlib` backend change. That backend change shouldn't be there, since we should be longer manually setting the backend after https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3420.

Reply:
> The CI error is from a matplotlib backend change, not because of PyQt.

I removed the `matplotlib` backend change in 92d5235, and _now_ the tests are [properly failing](https://github.com/spinalcordtoolbox/spinalcordtoolbox/runs/3048484333#step:7:26) because PyQt can't be imported on a headless system.

Reply:
> I'll see if there's a way to set PyQT to "headless mode" so to speak.

[Aha!](https://stackoverflow.com/a/55442821/7584115)

Reply:
Curses, [no such luck](https://github.com/spinalcordtoolbox/spinalcordtoolbox/runs/3048345891#step:5:30).

Maybe it's best to just disable these tests for CI, then.

Reply:
Famous last words, but _hopefully_ everything should be [fixed in our test suite](https://github.com/spinalcordtoolbox/spinalcordtoolbox/actions/runs/1023558778) by c22c23c.

Reply:
> Famous last words, but _hopefully_ everything should be [fixed across our full test suite](https://github.com/spinalcordtoolbox/spinalcordtoolbox/actions/runs/1023558778) by [c22c23c](https://github.com/spinalcordtoolbox/spinalcordtoolbox/commit/c22c23ca0ae96f44ea11057d369acfbb8fa0a87e).

So close. Everything worked except WSL. I'll rebase a fix for that, too.